sections/appendix/cat_auth_pachinko.tex

```diff
@@ -19,8 +19,16 @@ \subsection{Category and Author Pachinko}\label{app:cat_auth_pachinko}
 		Taxonomy \acrlong{pam} & 0.660 \\
 		\textbf{Author \acrlong{pam}} & \textbf{0.598} \\
 		\textbf{Category \acrlong{pam}} & \textbf{0.585} \\
+		\textbf{\acrlong{pam}} & textbf{0.670}
 	\end{tabular}
 	\label{tab:cat_auth_pachinko_results}
 \end{table}
 
-\langballe[inline]{paragraph about comparison to normal pachinko with no metadata and no locking (90-90 layers)}
+For comparison we also ran a four level \gls{pam} without any metadata, using 100 and 90 as the sizes of the two middle layers.
+This ended up providing very good results, slightly better than the Taxonomy \gls{pam}.
+This also shows that while Category \gls{pam} and Author \gls{pam} get better results than their \gls{lda} counterparts, it is better to run \gls{pam} without modifications.
+This could be due to our application, but it might also point towards these two particular metadata fields not being particularly useful in this specific application.
```

Comment:
Hvad menes der specifikt med "application"?

Reply:
jeg mente at du kunne bare være fordi at vores metoder til at incorporere metadata er skrald

Reply:
```suggestion
This could be due our models being unable to make good use of the extra information provided by the metadata, but it might also point towards these two particular metadata fields not being particularly useful in this specific project.
```

Reply:
```suggestion
This could be due to our models being unable to make good use of the extra information provided by the metadata.
It might also point towards these two particular metadata fields not being particularly useful in this specific project.
```